### DISCUSSION TITLE
As shown in the picture, what is the problem?

### DISCUSSION THREAD

**Discussion by visiLi** [floccusaddon/floccus#1512] [Q&A] [answered]
![20240105210352](https://github.com/floccusaddon/floccus/assets/42717403/93da0993-0a56-42f4-918f-bd1741a13f9e)

**Comment by marcelklehr** [comment, marked as answer]
Hi! Floccus will repeatedly try to sync, while the sync is in "scheduled" state, but this means that the server resource is currently locked by a sync process from a different client. If you are positive that no other clients are currently syncing, you can manually remove the lock file on the webdav server.

**Reply by visiLi** [reply]
Control options will be added later, self-service control whether to lock the file

**Reply by marcelklehr** [reply]
I'm sorry, I don't understand. You can also write in your mother tongue, if that's easier :)

**Reply by visiLi** [reply]
后续会添加开关选项 用于控制是否锁定吗
